index.js

```diff
@@ -33,4 +33,5 @@ exports.got = require('./lib/got')
 exports.linewrap = require('./lib/linewrap')
 exports.Spinner = require('./lib/spinner')
 exports.exit = require('./lib/exit').exit
-exports.login = require('./lib/login')
+exports.auth = require('./lib/auth')
+exports.login = require('./lib/auth').login
```

Comment:
@dickeyxxx might be better written as `exports.login = exports.auth.login` so that you do not have to go through the require again